src/loaders/compute_tools/checkm2/checkm2.py

```diff
@@ -47,12 +50,23 @@ def _run_checkm2(
           )
 
     tool_file_name, genome_id_col = 'quality_report.tsv', 'Name'
-    process_genome_attri_result(output_dir,
-                                SELECTED_CHECKM2_FEATURES,
-                                genome_id_col,
-                                ids_to_files,
-                                [tool_file_name],
-                                )
+    genome_attri_docs = process_genome_attri_result(output_dir,
+                                                    SELECTED_CHECKM2_FEATURES,
+                                                    genome_id_col,
+                                                    ids_to_files,
+                                                    [tool_file_name],
+                                                    )
+
+    # create fatal error file if any genome is missing from the result file
+    fatal_tuples = []
+    error_message = f"Result missing in the CheckM2 output file."
```

Comment:
is it not possible to be more specific than this?

Reply:
👍

Reply:
So if checkm fails it just leaves a blank spot in the report? Are there any logs we could look at to get a better idea of the problem?

Reply:
yea. The result file just miss one of the genomes. And checkm finished fine with no errors. I don't recall I saw anything in the logs. But after rerunning we lost logs from the previous one. I found out the bug upon running the parser script.

Reply:
Ok, I guess this is the best we can do for now then. If we start seeing a genome that consistently causes a fail we can look again and see if we can improve the error message

Reply:
:+1: